app/jmeter/jira.jmx

```diff
@@ -9611,6 +9611,7 @@ if ( sleep_time &gt; 0 ) {
             <assertionsResultsToSave>0</assertionsResultsToSave>
             <bytes>true</bytes>
             <sentBytes>true</sentBytes>
+            <url>true</url>
```

Comment:
Do you know what is this about?

Reply:
Yes, I have found out what is it 
![image](https://user-images.githubusercontent.com/30836532/117790682-43d49600-b252-11eb-9e78-24ac1ec43a3c.png)